lemur/certificates/service.py

```diff
@@ -864,3 +864,13 @@ def cleanup_after_revoke(certificate):
 
     database.update(certificate)
     return error_message
+
+
+def get_issued_cert_count_for_authority(authority):
+    """
+    Returns the count of certs issued by the specified authority.
+
+    :return:
+    """
+    query = database.session_query(Certificate.id).filter(Authority.id == authority.id)
```

Comment:
I think I have found the reason behind 40 issued certs: 
```
Certificate.authority_id == authority.id
```

This does something close to what we need here, but not at the level of CA, but the issuer type (like all acme-issuer based CAs):

https://github.com/Netflix/lemur/blob/56169656377637b7a0e53514105fbe31b436c4a9/lemur/certificates/service.py#L118-L123

in the Lemur UI, the Authority page renders all certs issued off a CA, and I was looking for that functionality, when I came across the code behind it: 

https://github.com/Netflix/lemur/blob/37e58574066921f50485398d195994b1b805c216/lemur/authorities/views.py#L492-L517

the Authority keeps track off all the children ever issued:
https://github.com/Netflix/lemur/blob/37e58574066921f50485398d195994b1b805c216/lemur/authorities/models.py#L54-L56

Reply:
Thanks, good catch. I'm finding it a bit difficult to locate good examples for queries for SQLAlchemy 😅  but I think it should be correct now.